src/components/Button/Button.js

```diff
@@ -0,0 +1,8 @@
+import van from 'vanjs-core'
+import '../../style.css'
+import './button.css'
+
+const { button } = van.tags
+
+export const Button = ({ text, onClick, type }) =>
+  button({ onclick: onClick, class: `button ${type || 'default'}` }, text)
```

Comment:
@zr0z I added these params except for `value`. What do you mean by this? There is already a `text` passed to this component

Reply:
@Ania-chan by default `button` is assumed to act on a form, so the behavior of the button should be in a `value` property and it will be sent to the server alongside the `name`.

If you assume that your component will never be used within a form then it should have a [`type` set to `button`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type).

From MDN:

> If your buttons are not for submitting form data to a server, be sure to set their type attribute to button. Otherwise they will try to submit form data and to load the (nonexistent) response, possibly destroying the current state of the document.